dev-docs/modules/liveIntentRtdProvider.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: page_v2
+title: LiveIntent RTD Module
+display_name: LiveIntent RTD Module
+description: LiveIntent Real Time Data Module
+page_type: module
+module_type: rtd
+module_code : liveIntentRtdProvider
+enable_download : true
+vendor_specific: true
+sidebarType : 1
+---
+
+# LiveIntent RTD Module
+
+{% include dev-docs/loads-external-javascript.md %}
+
+This module extracts segments from `bidRequest.userId.lipb.segments` enriched by the userID module and

```

Comment:
This is a good technical description, but it would be best to start out with a product-oriented description. Why would a business person care about `lipb.segments`?

Reply:
You're right, I updated it